superchain/superchain-version_test.go

```diff
@@ -0,0 +1,61 @@
+package superchain_test
```

Comment:
Note new package to allow us to import from the monorepo without incurring a dependency cycle (nothing will import this code).